pkg/aws/iam.go

```diff
@@ -4,15 +4,21 @@ package aws
 
 import (
 	"context"
+	"log"
 	"net/url"
 	"reflect"
 
 	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/stretchr/testify/assert"
 	"gopkg.in/square/go-jose.v2/json"
 
 	"testing"
 )
 
+type IAMClient interface {
```

Comment:
**Chore:** Would you please put a comment here about what this is for? See various other files for examples.

Reply:
done